packages/log/src/LogExport.ts

```diff
@@ -109,6 +125,7 @@ export async function exportLogs(
   metadata?: Record<string, unknown>,
   reduxData?: Record<string, unknown>,
   blacklist: string[][] = DEFAULT_PATH_BLACKLIST,
+  maximumDepth: number = DEFAULT_MAXIMUM_DEPTH,
   fileNamePrefix = `${formatDate(new Date())}_support_logs`
```

Comment:
Changing the order of the parameters is a breaking change, put it on the end.
```suggestion
fileNamePrefix = `${formatDate(new Date())}_support_logs`,
  maximumDepth: number = DEFAULT_MAXIMUM_DEPTH
```

Reply:
In most cases it seems to be the `configs` object under `client` or `dheClient`, e.g. `schemaService.clientUtils.client.configs`, with each key value pair in that map being serialized

<img width="389" height="431" alt="Screenshot 2025-10-07 at 8 58 57 AM" src="https://github.com/user-attachments/assets/35c0e8fc-8298-4113-b12d-41c8d17d0f74" />